website/docs/d/team.html.markdown

```diff
@@ -34,3 +34,4 @@ data "github_team" "example" {
 * `permission` - the team's permission level.
 * `members` - List of team members (list of GitHub usernames). Not returned if `summary_only = true`
 * `repositories` - List of team repositories (list of repo names). Not returned if `summary_only = true`
+* `repositories_detailed` - List of team repositories (list of `repo_id` and [`role_name`](https://registry.terraform.io/providers/integrations/github/latest/docs/resources/team_repository#permission)). Not returned if `summary_only = true`
```

Comment:
Are there future commits incoming for the specified `summary_only` property? I don't see an implementation.

Reply:
No need, it's already there. The entire [L109-L192 block](https://github.com/integrations/terraform-provider-github/blob/ec307a35525e744f579fa64399f4f1bd2f8ab1a5/github/data_source_github_team.go#L109-L192) only happens if the `summary_only` is set to true. 

My implementation is more or less just a copy-paste of the `repositories` field, which already had the `summary_only` field implemented 😅

Reply:
Oh thank you, I missed that piece somehow!